docker/cloudbuild.yaml

```diff
@@ -47,7 +46,6 @@ substitutions:
     _CUDA_COMPUTE: '3.7,7.0,7.5,8.0'
     _GITHUB_XLA_BRANCH: ''
     _GITHUB_EXAMPLE_BRANCH: 'master'
-    _BUILD_CPP_TESTS: '1'
```

Comment:
So after we switch to bazel, do we build cpp test by default?

Reply:
Please see the comment in docker/Dockerfile.

We don't need to prebuild tests; they will be built during the test execution phase; most of the stuff will be cached because the largest chunk of the code is xla.

I used 3.8, 3.9, 3.10, 3.11 python versions over the time.